app/config.py

```diff
@@ -74,6 +74,8 @@ class Config(object):
     EXPIRY_WARNING_MINS = 58
     INACTIVITY_WARNING_DURATION = 2
 
+    ADMIN_ACTION_ALLOW_SELF_APPROVAL = True
+
     FUNCTIONAL_TEST_PERMANENT_SESSION_LIFETIME = 0.667 * 60
     FUNCTIONAL_TEST_INACTIVITY_MINS = 0.167
```

Comment:
We've had some major issues in the past with default on behaviour.  I would be tempted to default this to False and then pull it from the environment if it's set, or specifically set it to False only in the dev environment.

Reply:
Good thinking, I would prefer that too.